### PR TITLE
archive docs and point to new location

### LINKED ISSUE #1624: archive docs and point to new location
there has been some confusion with people trying to use these old and in some cases broken docs.. we need to archive docs/ and have them link over to the containerd/containerd/docs/cri path

### PR DESCRIPTION
**Pull request by zouyee**
fix https://github.com/containerd/cri/issues/1624

Signed-off-by: Zou Nengren <zouyee1989@gmail.com>

### CODE REVIEW

**Review by AkihiroSuda — APPROVED** [review]

**Review by Zyqsempai — APPROVED** [review]

File: docs/README.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# NOTE
+
+On October 7, 2020, the contents of this repo were merged into [the `containerd/containerd` repo](https://github.com/containerd/cri).
+Related docs previously stored under [`containerd/cri/docs`](https://github.com/containerd/cri/tree/release/1.4/docs)
+was moved to [`containerd/containerd/docs/cri` package](https://github.com/containerd/containerd/tree/main/docs/cri).
```
Comment (mikebrow) [inline docs/README.md:5]: s/was/were/

File: docs/architecture.md [hunk 2]
```diff
@@ -1,18 +0,0 @@
-# Architecture of The CRI Plugin
```
Comment (mikebrow) [inline docs/architecture.md:1]: If you just delete the files all old links will abend with 404 browser errors which isn't very helpful.

I was thinking these old md files would be come link only files with one link pointing to the last commit aka permanent link in GitHub for this 1.4/release... e.g. `https://github.com/containerd/cri/blob/f6026296a3991010429db91e7e677f9c9d4861ab/docs/architecture.md`

You can get that from the `https://github.com/containerd/cri/blob/release/1.4/docs/architecture.md` directory/file and choosing `...` - `Copy permalink` 

And the other link would of course point to the new location in containerd/containerd/docs/cri off the main branch saying something like for containerd 1.5 + this architecture document has been relocated.

Comment (mikebrow) [reply docs/architecture.md:1]: for convenience you could also put all the new and old links in the readme..

**Review by mikebrow — COMMENTED** [review]
see comments.. 

Thanks!

**Pull request closed without merging**